gapis/gfxapi/gles/replay.go

```diff
@@ -180,6 +180,9 @@ func (a api) Replay(
 	// Cleanup
 	transforms.Add(&destroyResourcesAtEOS{})
```

Comment:
IIRC there was a leak on some driver that required a device to be restarted after a large number of replays.